app/controllers/concerns/publishers/wizardable.rb

```diff
@@ -61,19 +68,17 @@ def job_details_params(params)
       status: vacancy.status.blank? ? "draft" : nil,
     }
     params.require(:publishers_job_listing_job_details_form)
-          .permit(:job_title, :contract_type, :contract_type_duration, subjects: [])
+          .permit(:job_title, :contract_type, :contract_type_duration, key_stages: [], subjects: [])
           .merge(attributes_to_merge.compact)
   end
 
   def working_patterns_params(params)
-    params.require(:publishers_job_listing_working_patterns_form)
-          .permit(:working_patterns_details, working_patterns: [])
+    params.require(:publishers_job_listing_working_patterns_form).permit(:working_patterns_details, working_patterns: [])
           .merge(completed_steps: completed_steps)
   end
 
   def pay_package_params(params)
-    params.require(:publishers_job_listing_pay_package_form)
-          .permit(:actual_salary, :salary, :benefits)
+    params.require(:publishers_job_listing_pay_package_form).permit(:actual_salary, :salary, :benefits)
```

Comment:
I preferred this to be on 3 lines instead of 2, but it's just personal preference